_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -538,6 +561,7 @@ const features = {
 	portfolio: {
 		mapStateToProps: state => {
 			return {
+				feature: 'portfolio',
 				title: __( 'Portfolio: Custom content types' ),
```

Comment:
We use a bit different copy here and in testimonials: https://github.com/Automattic/jetpack/pull/15914/files#diff-2f517b5b3d753aa8f00a29502e24947aR980